shell/platform/embedder/embedder.h

```diff
@@ -306,12 +306,26 @@ typedef bool (*TextureFrameCallback)(void* /* user data */,
                                      FlutterOpenGLTexture* /* texture out */);
 typedef void (*VsyncCallback)(void* /* user data */, intptr_t /* baton */);
 
+typedef struct {
+  /// The width of the surface that will be backed by the fbo.
+  uint32_t width;
+  /// The height of the surface that will be backed by the fbo.
+  uint32_t height;
+} FlutterFrameInfo;
+
+typedef uint32_t (*FBOWithFrameInfoCallBack)(void* /* user data */,
+                                             FlutterFrameInfo /* frame info*/);
+
 typedef struct {
   /// The size of this struct. Must be sizeof(FlutterOpenGLRendererConfig).
   size_t struct_size;
   BoolCallback make_current;
   BoolCallback clear_current;
   BoolCallback present;
+  /// This is an optional callback. Exactly one of
```

Comment:
Instead of saying this is optional, can we say "specifying one (and only one) of the `fbo_callback` or `fbo_with_frame_info_callback` is required. Specifying both is an error and engine intialization will be terminated.".

Reply:
Done